### ISSUE TITLE
Not installable on SLES11SP4

### ISSUE THREAD

**Issue by danielnelson** [influxdata/telegraf#2764]
### Relevant telegraf.conf:
N/A

### System info:

SLES11SP4

### Steps to reproduce:

1. sudo rpm -ivh telegraf-1.3.0~rc2.x86_64.rpm

### Expected behavior:

Install successfully

### Actual behavior:
```
error: Failed dependencies:
        rpmlib(TildeInVersions) <= 4.10.0-1 is needed by telegraf-1.3.0~rc2-1.x86_64
```

### Additional info:
N/A

**Comment by danielnelson** [comment]
Possibly caused by updating the circleci container.

**Comment by danielnelson** [comment]
Can be fixed by removing the ~ from the version.

**Comment by danielnelson** [comment]
OpenSUSE recommends to use tilde versions and warns about using the Release tag.  However tilde versioning is not an option for SLES11.  Furthermore this advice is in contradiction to Fedora guidelines.

https://en.opensuse.org/openSUSE:Package_naming_guidelines

> Use “tilde versions”. The spec's Version: field may contain a tilde as a special marker that sorts before anything else, including the end-of-string, that is, "1.8~" < "1.8". This feature is available starting from rpm-4.10 (and was backported into openSUSE 12.2's rpm 4.9.1). Use: Version: 1.8.0~beta2. Tildes can be used multiple times, should it really become necessary. 

> Do not attempt to abuse the rpm Release: tag for version information either. It is the wrong field to convey this information, and the Open Build Service will by default overwrite Release: anyway with checkin and build counters. 

Fedora recommends using the final version number as the Version and prerelease info in the Release tag:

https://fedoraproject.org/wiki/Packaging:Versioning

> Those items which are present are combined (with periods to separate them) to construct the final Release: tag. In the usual notation where square brackets indicate that an item is optional:
> 
>     <pkgrel>[.<extraver>][.<snapinfo>]%{?dist}[.<minorbump>] 

> Release and post-release versions
> In the Version: tag, use the version that upstream has determined the next release will be. For the <pkgrel> field of the Release: tag, use a number of the form "0.N" where N is an integer beginning with 1 and increasing for each revision of the package. Prerelease versions MUST use a Release: tag strictly less than 1, as this is the sole indicator that a prerelease has been packaged. 

Fedora has not yet accepted the proposal for tilde versioning, though it appears at this time that they may in the future:

https://pagure.io/packaging-committee/issue/398

**Comment by danielnelson** [comment]
My current plan, pending additional testing, is to use the Release tag as suggested by Fedora, we can switch to tilde versioning when/if it is better supported.

Prerelease:
```
$ rpm -qip build/telegraf-1.3.0-0.rc2.x86_64.rpm
Name        : telegraf
Version     : 1.3.0
Release     : 0.rc2
```

Final Release:
```
$ rpm -qip build/telegraf-1.3.0-1.x86_64.rpm
Name        : telegraf
Version     : 1.3.0
Release     : 1
```
